stress_test/controller.py

```diff
@@ -54,45 +56,41 @@ def __init__(self, ctrl_base_dir, test_config):
 
         self._ssh_conn = None
 
-        #check handlers' validity
+        '''check handlers' validity'''
```

Comment:
Don't use triple backticks for inline or block comments 

The convention is (https://www.python.org/dev/peps/pep-0008/#id29):
- use triple backticks (`'''`) for docstring comments only (at module level, class level, function level)
- use `#` for inline comments or block comments

Change this everywhere else you might have it